db/migrate/20200605100827_create_email_address_verifications.rb

```diff
@@ -0,0 +1,13 @@
+class CreateEmailAddressVerifications < ActiveRecord::Migration[6.0]
+  def change
+    create_table :email_address_verifications do |t|
+      t.string :email, null: false
```

Comment:
Postgres string type is case sensitive. `maciej@email.com` and `Maciej@email.com` are different records. I'd strongly suggest using `citext` extension in postgres to avoid unexpected behaviours and saving some time on formatting and querying the data.

Reply:
thanks, wilco

Reply:
Maybe that's not a huge deal - we do not interested in difference between `macjei@` and `Macieji@`, as long as its a vaild email.

Reply:
It is sort-of big deal in this sense that:
* You get duplicate records on different casing of the same email.
* You then have to perform the validation multiple times.

Most importantly for me, when you have to compare emails, for example for search or sign-on, you have to remember to downcase the text before continuing with the operation. Being forgetful, I always vote for some program doing a thing for me :)